crates/forge/src/lib.rs

```diff
@@ -145,6 +147,10 @@ pub struct TestArgs {
     /// Build contracts separately in the scarb starknet contract target
     #[arg(long)]
     no_optimization: bool,
+
+    /// Additional arguments for cairo-coverage or cairo-profiler
+    #[clap(last = true)]
+    additional_args: Vec<OsString>,
```

Comment:
Could you please describe shortly in pr summary why `allow_hyphen_values` and `last` were introduced?